network-api/networkapi/templates/fragments/byline.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n %}
+
+<div class="d-flex flex-wrap {% if show_full_info %}flex-column mb-4{% else %}align-items-center{% endif %}">
+  {% include "fragments/profile_images_stack.html" with profiles=authors %}
+
+  <p class="tw-text-sm my-2 tw-text-black">
+    {% spaceless %}
+      {% if show_full_info %}
+        {% trans "By" %}
+      {% endif %}
+
+      {% include "fragments/profile_names_list.html" with profiles=authors %}
```

Comment:
**Question- non blocking**

I'm not sure of the `show_full_info` check here... if we are rendering a name list don't we always want to show `By` ?

Reply:
Yea, interestingly no. The "full info" is used in page headers typically. There we show the "By" and also the publication date. Outside of that we often use the byline on cards where we want it a bit more compact and don't display the "By" nor the date.